admin/app/components/solidus_admin/ui/table/component.rb

```diff
@@ -136,6 +139,90 @@ def render_data_cell(cell, data)
     content_tag(:td, content_tag(:div, cell, class: "flex items-center gap-1.5"), class: "py-2 px-4 h-10 vertical-align-middle leading-none")
   end
 
+  def search_form_options
+    {
+      url: @search_page_url,
+      method: :get,
+      class: "h-14 p-2 bg-white border-b border-gray-10 relative hidden",
+      data: {
+        "#{stimulus_id}_target": "searchForm"
+      }
+    }
+  end
+
+  def search_field_options
+    {
+      class: "border-2 border-gray-100 bg-white h-10 px-10 rounded-lg text-sm focus:outline-none flex-grow",
+      placeholder: "Search",
```

Comment:
Should this go under `i18n`? Also, because the search is by `name_or_slug_cont`, I think we could add those fields in the placeholder, eg `Search by Name or Slug`, so maybe it's a good idea to create a i18n key the search fields configuration included. 

<img width="1159" alt="Screenshot 2023-07-26 at 10 13 40@2x" src="https://github.com/solidusio/solidus/assets/167946/83379668-ee11-48d6-9256-1d8cc34473d7">

On a side note, currently Solidus allows searching for Name and SKU with the text fields, shouldn't we reflect that?

Reply:
You're right. I've adjusted the search key by using the variant association, enabling the search bar to look up by product name or variants SKU.

As for the placeholder, we're now using an i18n key related to that specific search key:
<img width="1325" alt="Screenshot 2023-07-27 at 12 28 44" src="https://github.com/solidusio/solidus/assets/19948291/16826faa-34ad-4b3a-b593-3d0804feeb4b">

Thank you @kennyadsl and let me know what do you think 👍